src/backend/metal/src/command.rs

```diff
@@ -775,6 +776,19 @@ impl RawCommandQueue<Backend> for CommandQueue {
                 msg_send![render_layer, setContents: io_surface.obj];
             }
         }
+
+        if cfg!(debug_assertions)
+        {
+            unsafe {
+                if let Some(mtl_capture_manager) = Class::get("MTLCaptureManager")
```

Comment:
looks like we'll need some capture API in `metal-rs` to avoid raw obj-c handling here